squashfs/squashfs.go

```diff
@@ -299,5 +304,16 @@ func GuestMount(squashFile string, mountpoint string, rootHash string) error {
 	if err != nil {
 		return fmt.Errorf("Failed releasing squashfuse process: %w", err)
 	}
+
+	for !isMountpoint(extractDir) {
+		fmt.Printf("%s is not yet mounted...\n", extractDir)
+		time.Sleep(time.Second)
```

Comment:
thanks for this.. I thought I had dome something, but can't find it. 

1 second is an eternity here. this probably happens in the time frame of small milliseconds, a one second penalty for missing once seems way high to me.

As a simpler poll, could you do get a fileInfo before calling squashfuse, and then loop over a Stat and call to [os.SameFile](https://pkg.go.dev/os#SameFile) ?

that would seem lighter to me than parsing /proc/mounts repeatedly.

Reply:
Oh, I like that SameFile idea, thanks.